packages/runtime-common/realm.ts

```diff
@@ -127,6 +128,9 @@ export interface RealmAdapter {
 
 interface Options {
   deferStartUp?: true;
+  enableLocalRealm?: true;
+  indexHTML?: string;
```

Comment:
perhaps we should hoist this out of options

Reply:
this might be better to use as a callback so we can lazily get the index html contents--or maybe just a URL that we fetch when asked